app/webpacker/components/Panel/pages/CompetitorRegistrationPage/index.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import {
+  Form, Header, Message, Label, Table, Loader,
+} from 'semantic-ui-react';
+import { useQuery } from '@tanstack/react-query';
+import { IdWcaSearch } from '../../../SearchWidget/WcaSearch';
+import SEARCH_MODELS from '../../../SearchWidget/SearchModel';
+import { fetchJsonOrError } from '../../../../lib/requests/fetchWithAuthenticityToken';
+import { viewUrls } from '../../../../lib/requests/routes.js.erb';
+
+async function getCompetitions({ wcaId }) {
+  const { data } = await fetchJsonOrError(
+    viewUrls.persons.registrations(wcaId),
+  );
+  return data || {};
+}
+
+function CompetitorRegistrationPage() {
+  const [formValues, setFormValues] = useState({});
```

Comment:
Why are you creating a whole `formValues` object if your only purpose is to store a WCA ID? You might just as well do `const [wcaId, setWcaId] = useState();`...